docs/pages/tutorial/introduction.mdx

```diff
@@ -8,15 +8,15 @@ import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
 
 import { BoxLink } from '~/ui/components/BoxLink';
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
-import { SnackInline } from '~/ui/components/Snippet';
 
 We're about to embark on a journey of building universal apps. In this tutorial, we'll create a universal app that runs on Android, iOS, and the web; all with a single codebase. Let's get started!
 
 ## About this tutorial
 
 The objective of this tutorial is to get started with Expo and become familiar with the Expo SDK. It'll cover the following topics:
 
-- Create an Expo App
+- Create an app using the `default` template with TypeScript enabled
```

Comment:
Not sure "default" needs syntax highlighting, since it's not really syntax you put into the command.

Reply:
We use syntax highlight for the templates in Create Expo App reference: https://docs.expo.dev/more/create-expo/#--template, since it is considered an option, which is passed as an option to the command.

I've removed the inline syntax highlight from here and another instance in the next chapter.